src/opnsense/scripts/interfaces/traffic_top.py

```diff
@@ -94,15 +95,32 @@ async def request_ittr(self, addresses):
         dnsResolver.timeout = 2
         tasks = []
         for address in addresses:
-            tasks.append(dnsResolver.resolve_address(address))
```

Comment:
Why are you trying to re-implement the function of `resolve_address()` in the code below? According to the upstream documentation (https://dnspython.readthedocs.io/en/latest/resolver-class.html), `resolve_address()` should already return a valid response for both address families.

Can you offer some example addresses where the code below offers your expected outcome and `resolve_address()` doesn't?

Reply:
I wasn't trying to re-implement it. But the following part was hard-coded to work only with IPv4 (.in-addr.arpa). My goal was only to allow translation for any IP address that comes up.
I was surprised to find that low-level translation to PTR there and only expanded on it. If there is a better solution, I'll gladly accept it.

Reply:
Please offer examples of addresses where your code offers different outcomes than the existing code, https://github.com/rthalley/dnspython/blob/740ae72d28e5bdadafbda80169eea255bbde16ac/dns/asyncresolver.py#L113-L139 suggests it does work on both, digging deeper in the code (https://github.com/rthalley/dnspython/blob/740ae72d28e5bdadafbda80169eea255bbde16ac/dns/reversename.py#L31-L66) doesn't appear to do anything else than support both either....

Reply:
I guess you mean why not use this: 

```python
    async def request_ittr(self, addresses):
        self._results = {}
        try:
            dnsResolver = Resolver()
        except dns.resolver.NoResolverConfiguration:
            return
        dnsResolver.timeout = 2
        tasks = []
        for address in addresses:
            tasks.append(dnsResolver.resolve_address(address))

        responses = await asyncio.gather(*tasks, return_exceptions=True)

        for response, address in zip(responses, addresses):
            if isinstance(response, dns.resolver.Answer):
                if ":" in address:
                    addr = str(ipaddress.IPv6Address(address))
                else:
                    addr = ".".join(reversed(response.canonical_name.to_text().replace('.in-addr.arpa.', '').split('.')))

                for item in response.response.answer:
                    if isinstance(item, dns.rrset.RRset) and len(item.items) > 0:
                        self._results[addr] = str(list(item.items)[0])

        return self._results

``` 

I tried it, and in console it works fine. But on the web it just loops with an empty JSON response.

Reply:
with the risk of repeating myself, please offer example addresses for comparison.

Reply:
There isn't any difference that I know of.
I made a partial revert, would you like me to squash it?

Reply:
This isn't going to be merged, these commits look highly suspicious, without test examples (which I repeatedly asked for), we will stop our efforts in these PR's.